server/worker/main.py

```
@@ -0,0 +1,85 @@
+            out, err = run_docker_linter(
+                "pr-linter:latest",
+                temp_dir,
+                ["bash", "-c", "eslint " + " ".join(js_files) + " -f unix"]
+            )
+            if out:
+                findings.append("ESLint:\n" + "\n".join([line for line in out.splitlines()]))
+
+        # Markdown files
+        md_files = [f for f in changed_files if f.endswith(".md")]
+        if md_files:
+            out, err = run_docker_linter(
```

Ruff (0.13.2)

Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)